Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/model/MatchIdentifier.java

```diff
@@ -0,0 +1,75 @@
+package org.cqframework.cql.cql2elm.model;
+
+import org.apache.commons.lang3.tuple.Pair;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class MatchIdentifier {
```

Comment:
Should be IdentifierResolution

Reply:
Could reasonably have an `IdentifierMatch` class which was just the identifier, the matchType and the resolved Element, and then an `IdentifierResolution` class that collected the IdentifierMatches for a given identifier.